app/controllers/api/v1/on_duty_firms_controller.rb

```diff
@@ -13,13 +13,13 @@ def show
   def rota_slots_for_location
     RotaSlot.joins(:shift)
       .where(
-        shifts: { location_uid: requested_location },
```

Comment:
Yeah I'm a little out of whack with all of the names of stuff now. Technically a `Shift` belongs to an `Organisation` not a `Location`, as there's no differentiator atm.